ONSdigital/takeon-ui
# Feature/spp3479/audit user save responses

# Description
- Passed cognito user through to the Business layer for audit
- Some linting tidy up
Story link: https://collaborate2.ons.gov.uk/jira/browse/SPP-3479

# How to test?
Use env 3479 in dev.
Update responses and ensure the contributor table and response table have been updated. 

NOTE: To check the contributor table you may need to spin up locally as the system is the last to do an update in the table and won't show the user. You could also try and intercept what's going into the database before DA makes a change. Or run locally where DA doesn't run and check the DB.

For the response table the date questions and any that don't have DA etc run on them will have the username those that have DA etc run on them if will have `fisdba` in the lastupdatedby field.

If the user creates the responses the created by field in the db should show the username.

# Links
- There is data in the db for testing, but for testing loading of selection files please use: https://github.com/ONSdigital/spp-test-data-catalogue/tree/main/synthetic/e2e-test-v1/023-RSI
- Link to Business Layer PR: https://github.com/ONSdigital/takeon-business-layer/pull/97
